### PR TITLE
[BUILD] Default build against spark 3.1

### PR DESCRIPTION
**Pull request by pan3793**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,11 +29,10 @@ cache:
...
 matrix:
   include:
-    - name: Test Kyuubi w/ Default Profile
+    - name: Test Kyuubi w/ -Pspark-3.0 -Pspark-hadoop-2.7
       env:
-        - PROFILE=""
+        - PROFILE="-Pspark-3.0 -Pspark-hadoop-2.7"
         - JOB="build"
-        - CODECOV="true"
     - name: Test Kyuubi w/ -Pspark-3.0 and Spark 3.1 binary
```
Comment (yaooqinn) [inline .travis.yml:36]: this shall be -Pspark-3.1 and Spark 3.0 binary

File: pom.xml [hunk 2]
```diff
@@ -68,10 +68,10 @@
         <commons-collections.version>3.2.2</commons-collections.version>
         <commons-lang3.version>3.10</commons-lang3.version>
         <curator.version>2.12.0</curator.version>
-        <delta.version>0.8.0</delta.version>
+        <delta.version>1.0.0</delta.version>
```
Comment (yaooqinn) [inline pom.xml:71]: is this related？

Comment (pan3793) [reply pom.xml:71]: spark 3.1 needs delta 1.0.0

File: .travis.yml (hunk 1 above)
Comment (pan3793) [reply .travis.yml:36]: make sense

File: pom.xml (hunk 2 above)
Comment (yaooqinn) [reply pom.xml:71]: does delta 1.0.0 drop spark 3.0 support？

Comment (pan3793) [reply pom.xml:71]: Yes, see https://github.com/delta-io/delta/issues/690

Comment (yaooqinn) [reply pom.xml:71]: so this should be added to our 3.0 profile

Comment (pan3793) [reply pom.xml:71]: it's already there :)

**Comment by yaooqinn** [comment]
please fix the conflicts

**Comment by pan3793** [comment]
Conflicts have been solved, and also cc @turboFei @ulysses-you

**Review by yaooqinn — APPROVED** [review]
LGTM

**Review by turboFei — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**